extensions/ql-vscode/src/vscode-tests/minimal-workspace/sourcemap.test.ts

```diff
@@ -0,0 +1,71 @@
+import { fail } from 'assert';
+import { commands, Selection, window, workspace } from 'vscode';
+import * as path from 'path';
+import * as assert from 'assert';
+import { expect } from 'chai';
+import { tmpDir } from '../../helpers';
+import * as fs from 'fs-extra';
+
+export function run() {
+  /**
+   * Integration tests for queries
+   */
+  describe('SourceMap', function() {
+    this.timeout(200000);
+
+    it('should jump to QL code', async () => {
+      try {
+        this.timeout(60000);
```

Comment:
This is a little bit strange. `this` is the enclosing describe object. I am not sure if the `timeout()` is actually affecting this test.  I think you want to do this.

```suggestion
    it('should jump to QL code', async function() {
      try {
        this.timeout(60000);
```

Reply:
I removed that timeout anyway.